common/scripted_effects/00_murder_scheme_effects.txt

```diff
@@ -262,9 +262,11 @@ murder_failure_effect = {
 			add_dread = minor_dread_gain
 		}
 		show_as_tooltip = {
-			attempted_murder_opinion_effect = {
+			#Warcraft
+			attempted_hostile_scheme_opinion_effect = {
 				VICTIM = scope:target
 				MURDERER = scope:owner
```

Comment:
You pass `MURDERER`, but it accepts `ATTACKER`. Another opinion effect has the same problem.

Reply:
Will be fixed when Zum merges `scheme-opinion` branch

Reply:
Done!